### ISSUE TITLE
Warning on 1.11.2:  The Fetch API is an experimental feature

### ISSUE THREAD

**Issue by mk-git** [massimocandela/geofeed-finder#30]
The following warning message is displayed in the latest version 1.11.2. The `--trace-warnings` gives no further output. Just wanted to report this for your information.

I have tested it with the release of the two different versions under Linux (Debian 12) and Windows 10

Version 1.11.1:
```
./geofeed-finder-linux-x64.1 -t 87.245.68.194
https://www.as35518.net/geofeed.csv [download]
```

Version 1.11.2:
```
./geofeed-finder-linux-x64 -t 87.245.68.194
https://www.as35518.net/geofeed.csv [download]
(node:1105474) ExperimentalWarning: The Fetch API is an experimental feature. This feature could change at any time
(Use `geofeed-finder-linux-x64 --trace-warnings ...` to show where the warning was created)
```

Version 1.11.2 with trace-warning:
```
./geofeed-finder-linux-x64 --trace-warning -t 87.245.68.194
https://www.as35518.net/geofeed.csv [download]
(node:1105630) ExperimentalWarning: The Fetch API is an experimental feature. This feature could change at any time
(Use `geofeed-finder-linux-x64 --trace-warnings ...` to show where the warning was created)

```

**Comment by sid6mathur** [comment]
My understanding is the releases v1.11.2 and v1.11.1 are producing zero-sized output CSV due to an error in the initial WHOIS data download step. v1.11.0 does NOT demonstrate this issue. I have been unable to identify the cause. 

Evidence: the CSV artifact published on geolocatemuch.com is zero-sized as of today: https://geolocatemuch.com/geofeeds/validated-all.csv

**Comment by massimocandela** [comment]
Hi all, the warning reported by @mk-git is just a warning and can be safely ignored. Fetch is now heavily used in node and the features I'm using are well supported.

The issue reported by @s8mathur is not related to the fetch. It was related two 2 separated issues: (1) ARIN temporarily (probably a glitch) changed the file name of their stat files, and (2) a failed download in that stage was resulting in a crash. Both issues solved in v1.2.0.

**Comment by sid6mathur** [comment]
Hi @massimocandela ; how long does a single run of the tool take these days? Using no additional flags, for example.
